tests/subyt/test_conditional.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import time
+import unittest
+from pathlib import Path
+
+import pandas as pd
```

Comment:
apparently this test (only) is introducing dependency to pandas
--> poetry / project.toml should declare this as a test-only dependency ?